packages/react-kit/src/stories/Header.stories.tsx

```diff
@@ -0,0 +1,146 @@
+'use strict';
+import { Button } from '@components/Button';
+import { Header as NavHeader } from '@components/Header';
+import { Row } from '@components/Row';
+import PantheonLogo from '@stories/assets/pantheon-fist-blk.png';
+import type { Meta, StoryObj } from '@storybook/react';
+import clsx from 'clsx';
+
+const Logo = () => {
+	return (
+		<a href="/">
+			<img
+				src={PantheonLogo}
+				width="40"
+				height="40"
+				alt="Pantheon Systems"
+				className="lg:rk-mr-8"
```

Comment:
We'll have to talk through this with Cindy, but seeing this in browser I'm not convinced the nudge of margin the design gives to the logo and the icon buttons works. It makes things look misaligned to me. I'd expect to have a similar feeling when we start to see this in page layouts.

<img width="425" alt="logo_alignment" src="https://github.com/pantheon-systems/decoupled-kit-js/assets/889478/f72305b4-814c-459b-a253-da6f2b657afb">

Reply:
I also wonder about how much of things like pixel widths and tailwind alignment should be the responsibility of the consumer here. It's a tradeoff though - the more we own, the more likely they will get it right, but it will also result in less control for the user.